airflow/ui/src/pages/Dag/Overview/Overview.tsx

```diff
@@ -93,6 +100,15 @@ export const Overview = () => {
           startDate={startDate}
         />
       </HStack>
+      <SimpleGrid columns={3} gap={5} my={5}>
+        <Box borderRadius={4} borderStyle="solid" borderWidth={1} p={2}>
+          {isLoadingRuns ? (
+            <Skeleton height="200px" w="full" />
+          ) : (
+            <RunDuration runs={runs?.dag_runs} totalEntries={runs?.total_entries ?? 0} />
```

Comment:
If there are no runs, we should display a message like "No runs for this date range"